gym/wrappers/__init__.py

```diff
@@ -1,5 +1,13 @@
 """Module of wrapper classes."""
 from gym import error
+
+try:
+    import jumpy
+
+    from gym.dev_wrappers.lambda_reward import ClipRewardsV0, LambdaRewardV0
+except ImportError:
+    pass
```

Comment:
What happens when this fails but the user tries to import the class? 
Is the error helpful?

Reply:
Good catch, maybe raising a warning telling the user that all dev_wrappers will not work correctly since jumpy is not available for python3.6 would be better?

Reply:
I think that python 3.6 will be removed for the new wrappers

Reply:
If python3.6 supports removal is programmed anytime soon I can wait and then remove the whole try - except block from all the dev_wrappers PRs, what do you think? Removing the try-except block now will cause python3.6 pipelines to fail because of jumpy